v1/authorization/storage_authorization_test.go

```diff
@@ -225,6 +226,102 @@ func TestAuth(t *testing.T) {
 	}
 }
 
+func TestAuthBucketNotExists(t *testing.T) {
+	store := inmem.NewKVStore()
+	if err := all.Up(context.Background(), zaptest.NewLogger(t), store); err != nil {
+		t.Fatal(err)
+	}
+
+	ts, err := NewStore(store)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	bucketID := platform.ID(1)
+
+	tenant := tenant.NewStore(store)
+	err = tenant.Update(context.Background(), func(tx kv.Tx) error {
+		err := tenant.CreateBucket(context.Background(), tx, &influxdb.Bucket{
+			ID:    bucketID,
+			OrgID: platform.ID(10),
+			Name:  "testbucket",
+		})
+		if err != nil {
+			return err
+		}
+
+		b, err := tenant.GetBucketByName(context.Background(), tx, platform.ID(10), "testbucket")
+		if err != nil {
+			return err
+		}
+
+		bucketID = b.ID
+
+		return nil
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	perm1, err := influxdb.NewPermissionAtID(
+		bucketID,
+		influxdb.ReadAction,
+		influxdb.BucketsResourceType,
+		platform.ID(10),
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	perm2, err := influxdb.NewPermissionAtID(
+		platform.ID(2),
+		influxdb.ReadAction,
+		influxdb.BucketsResourceType,
+		platform.ID(10),
+	)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Why is this call to Fatal so terse, where you provide more information in later calls?

Reply:
I wouldn't really expect this call to ever error, since the error is only whether its a valid permission or not, vs the later calls that are actually relevant to the business logic that we're trying to test.